### ISSUE TITLE
Build issue with Unity2017.4.33

### ISSUE THREAD

**Issue by tolosafan** [cunum/unity-osx-notarize#1]
Hello,
First: very good idea to create such a plugin. We hate Apple plumberies to be able to run an app on a computer..So thanks and congrats!
Second:
After installing on U3D 2017.4.33 
-unzip the github package
-Drag-n-drop all the Editor folder content from unity-osx-notarize-master/Assets/Editor to Editor of our project
We got these errors:
_Assets/Editor/Notarization/NotarizationProcessor.cs(7,25): error CS0234: The type or namespace name `Reporting' does not exist in the namespace `UnityEditor.Build'. Are you missing an assembly reference?_
_Assets/Editor/Notarization/NotarizationProcessor.cs(13,41): error CS0246: The type or namespace name `IPostprocessBuildWithReport' could not be found. Are you missing an assembly reference?_

Any idea? 
Thanks,

**Comment by cunum** [comment]
Hello,

yeah the plugin uses the IPostprocessBuildWithReport interface for post build automatic notarization, this API was introduced in Unity 2018.1, so it doesn't work with Unity versions prior to that. I just pushed a backwards compatible fix, let me know if this does work for you, can't test it myself I'm already on 2019.3.

Best regards

**Comment by tolosafan** [comment]
Many Thanks for the effort.
The previous error has disappeared.
But it remains:
_"Assets/Editor/Notarization/NotarizationProcessor.cs(68,33): error CS0246: The type or namespace name `BuildReport' could not be found. Are you missing an assembly reference?"_

**Comment by cunum** [comment]
Oh yeah, I missed two occurrences, you can pull again, it should work now.

**Comment by tolosafan** [comment]
Yeah, it builds now. Great & thanks! Let's do some tests with Apple's plumbery..
